tests/e2e/run-scanner-v4.bats

```diff
@@ -68,17 +68,167 @@ setup() {
     fi
 
     test_case_no=$(( test_case_no + 1))
+
+    export MAIN_IMAGE_TAG=${MAIN_IMAGE_TAG:-}
+    info "Overriding MAIN_IMAGE_TAG=$MAIN_IMAGE_TAG"
+
+    export ROX_SCANNER_V4=true
 }
 
 teardown() {
-    local namespaces=( "stackrox" "$CUSTOM_CENTRAL_NAMESPACE" "$CUSTOM_SENSOR_NAMESPACE" )
-    for namespace in "${namespaces[@]}"; do
-        if kubectl get ns "${namespace}" >/dev/null 2>&1; then
-            run remove_existing_stackrox_resources "${namespace}"
+    local central_namespace=""
+    local sensor_namespace=""
+    local namespaces=( )
+
+    if "${ORCH_CMD}" get ns "stackrox" >/dev/null 2>&1; then
+        central_namespace="stackrox"
+        sensor_namespace="stackrox"
+        namespaces=( "stackrox" "${namespaces[@]}" )
+    fi
+    if "${ORCH_CMD}" get ns "${CUSTOM_CENTRAL_NAMESPACE}" >/dev/null 2>&1; then
+        central_namespace="${CUSTOM_CENTRAL_NAMESPACE}"
+        namespaces=( "${central_namespace}" "${namespaces[@]}" )
+    fi
+    if "${ORCH_CMD}" get ns "${CUSTOM_SENSOR_NAMESPACE}" >/dev/null 2>&1; then
+        sensor_namespace="${CUSTOM_SENSOR_NAMESPACE}"
+        if [[ "${central_namespace}" != "${sensor_namespace}" ]]; then
+            namespaces=( "${sensor_namespace}" "${namespaces[@]}" )
         fi
+    fi
+
+    if [[ -z "${BATS_TEST_COMPLETED:-}" ]]; then
```

Comment:
It is not clear in documentation but do you know if skipped test considered as completed? I believe we don't need debug info from skipped tests. There is a `BATS_TEST_SKIPPED` env. If skipped tests are not marked as competed then we should filter them out using `BATS_TEST_SKIPPED`

Reply:
good point, will make sure to not print the analysis info when the test was skipped.